Dockerfile

```diff
@@ -1,7 +1,40 @@
-FROM domerd/marine-core:marine
+ARG MARINE_CORE_TAG=marine
 
-RUN python3.8 -m pip install tox
+FROM domerd/marine-core:$MARINE_CORE_TAG as builder
 
 WORKDIR /marine-python
 
-CMD tox
+COPY setup* ./
+COPY README* ./
+COPY LICENSE ./
+COPY marine ./marine
+
+ENV PY="/opt/python/cp38-cp38/bin/python"
+
+RUN mkdir marine/.wslibs && cp /build/run/libmarine.so /build/run/lib*so.0 marine/.wslibs/ && $PY setup.py bdist_wheel --dist-dir /tmp
+
+WORKDIR /dist
+
+COPY scripts /scripts
+
+RUN /scripts/expose_auditwheel.sh / && $PY /scripts/modify_auditwheel_policy.py && /scripts/patch_auditwheel_recursive_dependency_bug.sh
+
+RUN auditwheel repair -w /dist /tmp/marine*.whl
+
+
+
+FROM centos/python-38-centos7
```

Comment:
Maybe you can have a separate `ARG PYTHON_VERSION` and use it for constructing both this and the marine image name/tag I suggested earlier

Reply:
Nah.
The wheel is not specific to a python version, at least not in a binary sense (only in the usual python syntax compatibility).

Reply:
abi fun yay